woocommerce/woocommerce
# [Experimental] Price Filter: Fix styling issue when Inline input fields is enabled

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #44755.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Navigate to Single Product template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. In the block inserter, search for "Product Filter" or browse the available blocks until you find them.
5. Click on the "Product Filter: Price (Beta)" block to add it to your content.
6. Select the Product Filter: Price (Beta) block in the blocks list.
7. In the sidebar to the right side, enable the "Inline input fields" setting within the Inspector Controls.
8. Make sure the block is displayed with the input fields at the sides of the price range slider.
9. Click on Save to save the changes.
10. Visit the Shop page (https://linktoyourstore.com/shop/).
11. Make sure the block is displayed with the input fields at the sides of the price range slider.

| Before | After |
|--------|--------|
| ![CleanShot 2024-02-29 at 18 22 34@2x](https://github.com/woocommerce/woocommerce/assets/20469356/b6236d54-177d-44c2-82f6-6316cecc0d87) | ![CleanShot 2024-03-05 at 10 40 52@2x](https://github.com/woocommerce/woocommerce/assets/20469356/ef327f68-85a8-41c0-a7c8-017358544c7d) | 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Fix styling issue for the Price Filter block preventing fields from appearing inline when the Inline input fields option is enabled

#### Comment 

</details>
